### DISCUSSION TITLE
pglet vs flet

### DISCUSSION THREAD

**Discussion by ssinghm32** [flet-dev/flet#222] [General]
Hi @FeodorFitsner 

What's the relationship between pglet.io and flet.dev?

Is Flet a fork of pglet?  

Can you share more insight between these 2 frameworks?

Thanks.

**Comment by FeodorFitsner** [comment]
So, Flet is a successor to Pglet. Pglet was for web only, based on React and [Fluent UI](https://developer.microsoft.com/en-us/fluentui#/controls/web) library. While working on Pglet I realized:

1. It's "too little, too late" - there is a lot of well-established web GUI libraries for Python: Wave, Streamlit, etc.
2. Fluent UI library from Microsoft was a wrong bet. It only looked like an open-source, but in reality it's a library for internal Microsoft projects with "published source" where nobody cares about 3rd-party issues or contributions.
3. Web-based UI was a showstopper for desktop. To get with web something to a desktop your option is either Electron or WebView. WebView on macOS is Safari, but on Windows it depends on Windows version: "old" Edge or WebView2 which requires a separate download.
4. No way to mobile devices.
5. Last but not least - I didn't like the name "pglet" :)

When I saw Flutter announcements of Windows support, improved web support, their 2022 roadmap I was sold almost instantly :)
So, Flet is a spinoff (or fork if you will) from Pglet based on Flutter. Flutter is awesome, gives similar "picture" on all platforms with a minimal size/overhead and has a fantastic library of widgets! I see a strong commitment from Google and I confident in Flutter's future on all platforms.
